deployments/nasa/image/binder/environment.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
   - h5netcdf=0.7
   - dask=1.1
```

Comment:
for deployments using `onbuild` we could also remove any packages that are the same as pangeo-stacks/base-notebook or pangeo-notebook

Reply:
yes, let's leave that for a follow up PR.